### DISCUSSION TITLE
Reason for metallic sound in Dutch model?

### DISCUSSION THREAD

**Discussion by invenix** [coqui-ai/TTS#1218] [General Q&A]
Thanks for the amazing work on Coqui TTS! I have a quick question I was wondering about. I noticed that the Dutch model has a metallic kind of pronunciation while the others sound great. What is the reason for this?
See for example
https://mbarnig.github.io/TTS-Models-Comparison/

**Comment by erogol** [comment]
the model has been trained by one of our contributors @r-dh

so maybe he could comment on that

**Comment by r-dh** [comment]
Hi, it's been a while, so I can't comment on specifics, although I remember the final result was certainly not that bad.

Maybe my [dataset repository](https://github.com/r-dh/dutch-vl-tts) can help you further, it contains the original audio files, the training config (which could use improvements), two trained models and three examples in the Google Drive folder of how the model sounds with the default vocoder (Griffin-Lim?) which was provided at the time.

I hope this can help you somewhat, but if you have anymore specific questions, let me know.

**Comment by invenix** [comment]
Thanks for your responses! I will look into it
